src/styles/blocks/star.scss

```diff
@@ -0,0 +1,15 @@
+.stars {
+  display: flex;
+  font-size: 10px;
+
+  .stars_star {
+    background: url(../images/star-active.svg) no-repeat center;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  }
+
```

Comment:
I don't see styles for stars modifier (stars--4, for example)

Reply:
you should manage the number of active stars with modifiers

Reply:
you didn't add styles for other modifiers (if I change the modifier on stars--3, for example, the number of active stars should change)